packages/jaeger-ui/src/components/DeepDependencies/SidePanel/index.test.js

```diff
@@ -145,7 +145,7 @@ describe('<SidePanel>', () => {
           operation: testVertex.operation,
           service: testVertex.service,
         }),
-        expect.anything()
+        undefined
```

Comment:
can you elaborate on this change, as I see it in many places? Why wouldn't `expect.anything()` match `undefined` too?